app/src/main/java/com/github/fribourgsdp/radio/CreateQRCodeFragment.kt

```diff
@@ -0,0 +1,112 @@
+package com.github.fribourgsdp.radio
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+import android.content.Intent
+import android.graphics.Bitmap
+import android.graphics.Point
+import android.os.Bundle
+import android.text.TextUtils
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.Toast
+import androidmads.library.qrgenearator.QRGContents
+import androidmads.library.qrgenearator.QRGEncoder
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.os.bundleOf
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.setFragmentResult
+import com.google.zxing.WriterException
+
+
+class CreateQRCodeFragment(val ctx: Context, val lobbyId: Long ): DialogFragment() {
+    // variables for imageview, edittext,
+    // button, bitmap and qrencoder.
+    private lateinit var qrCodeIV: ImageView
+    private lateinit var rootView : View
+
+    private lateinit var cancelButton: Button
+    lateinit var bitmap: Bitmap
```

Comment:
I think `bitmap`doesn't need to be extracted.

Reply:
Yes, you are right